src/libecalc/dto/result/tabular_time_series.py

```diff
@@ -18,7 +19,7 @@
 
 
 class TabularTimeSeries(ABC, EcalcResultBaseModel):
-    name: str
+    name: Optional[str] = None
```

Comment:
Will setting name to optional create problems elsewhere?

Reply:
It might, `to_dataframe` assumes name is set when creating the column names.

Reply:
Changed, it is no longer optional.